src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -149,6 +150,20 @@ void PasswordGeneratorWidget::setStandaloneMode(bool standalone)
     }
 }
 
+void PasswordGeneratorWidget::keyPressEvent(QKeyEvent* e)
+{
+    if (!e->modifiers() || (e->modifiers() & Qt::KeypadModifier && e->key() == Qt::Key_Enter)) {
+        if (e->key() == Qt::Key_Escape && m_standalone == true) {
+            emit dialogTerminated();
+        } else {
+            e->ignore();
+        }
+    }
+    else {
```

Comment:
Please put this on the same line as the closing brace. Other than that, I think this patch is fine as is.